daemon/container_unix.go

```diff
@@ -797,7 +797,7 @@ func (daemon *Daemon) updateNetwork(container *Container) error {
 	return nil
 }
 
-func (container *Container) buildCreateEndpointOptions(n libnetwork.Network) ([]libnetwork.EndpointOption, error) {
+func (container *Container) buildCreateEndpointOptions(n libnetwork.Network, controller libnetwork.NetworkController) ([]libnetwork.EndpointOption, error) {
```

Comment:
Can we avoid passing the `libnetwork.controller` and simply check for ` n.Name() == "bridge"`
The bridge string has been anyway hard-coded all over this file.
Another PR should take care of replacing those with a daemon constant.